### PR TITLE
Scaling UI

### LINKED ISSUE #413: [Enhancement] Scaling UI to different device sizes
## Issue Description:
The entire padding, margin and white space (mostly through the sized box) contains a constant value. While building the screen, those values give a better outlook of UI on developer device (having a given height & width) but as the device changes (i.e height & width change) those constant space, in general, doesn't give a better UI outlook.

## Issue Severity:
Medium

## Observed Behavior:
Ui doesn't look effective for large device size.

## Expected Behavior:
The UI should be capable to automatically scale itself up based on device size

## Steps to reproduce the issue:
Launch app on screen size
1. 4.0"
2. 5.5"
2. 5.6"
3. 6.0"
4. 6.3"

You will notice the device UI outlook destruction.

## Does this issue need immediate attention?
Yes

## Are you willing to work on this issue:
Yes, this issue seems very interesting to me and I would like to devotee my time here.

### PR DESCRIPTION
**Pull request by sumitra19jha**
**What kind of change does this PR introduce?**
Closes #413 

**Did you add tests for your changes?**
Not required

**If relevant, did you update the documentation?**
Yes

**Summary**
This will enable the scaling of ui based on device

**Does this PR introduce a breaking change?**
No

**Other information**

https://user-images.githubusercontent.com/35868598/115969367-3c2aa700-a55a-11eb-9a39-e24b512b3d5d.mp4

### CODE REVIEW

File: test/widget_tests/set_url_page_test.dart [hunk 1]
```diff
@@ -176,7 +178,7 @@ void main() {
       /// Enter [calico.palisadoes.org] in [TextFormField].
       await tester.enterText(
         find.byType(TextFormField),
-        'calico.palisadoes.org',
+        'talawa-graphql-api.herokuapp.com/graphql',
```
Comment (palisadoes) [inline test/widget_tests/set_url_page_test.dart:181]: This was long overdue

**Comment by palisadoes** [comment]
@sumitra19jha Please fix the merge conflict

**Comment by sumitra19jha** [comment]
@palisadoes Conflict resolved

File: test/widget_tests/set_url_page_test.dart (hunk 1 above)
Comment (sumitra19jha) [reply test/widget_tests/set_url_page_test.dart:181]: Yes.

**Review by Sagar2366 — APPROVED** [review]

**Pull request merged**